ci/win32/typecheck.cmd

```diff
@@ -0,0 +1,9 @@
+
+pushd .
+cd /D "%~dp0"
+
+cd ..\..\
+call .\venv\Scripts\activate
+set PYTHONPATH=.\src\;%$PYTHONPATH%
+python -m mypy ./src/rtctools_heat_network ./tests/
```

Comment:
When I run this locally I only get the errors of the second folder specified, in this case the ./tests/, is that intentional?

Also we should turn off this test for now.